### DISCUSSION TITLE
cookie consent like https://github.com/jazzband/django-cookie-consent

### DISCUSSION THREAD

**Discussion by BrentHuang** [fastapi/fastapi#11954] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
nothing
```

### Description

cookie consent like https://github.com/jazzband/django-cookie-consent

or

integrated with https://github.com/orestbida/cookieconsent

### Operating System

Linux

### FastAPI Version

N/A

### Pydantic Version

N/A

### Python Version

N/A

**Comment by RedYaafte** [comment]
You can try using:

```python
from fastapi import FastAPI, Response

app = FastAPI()


@app.post("/cookie-and-object/")
def create_cookie(response: Response):
    response.set_cookie(key="fakesession", value="fake-cookie-session-value")
    return {"message": "Come to the dark side, we have cookies"}
```
